### PR TITLE
 Remove key from badge & upgrade pre-commit hooks

### PR DESCRIPTION
**Pull request by soerenmartius**

### CODE REVIEW

**Comment by soerenmartius** [comment]
> this PR destroys the already complete documentation...

fixed in 39c97f8

File: README.md [hunk 1]
```diff
@@ -163,58 +177,69 @@ for details and use-cases.
...
 - **`policy`**: The `aws_iam_policy` object.
 - **`policy_attachment`**: The `aws_iam_policy_attachment` object.
 
 ## External Documentation
+
 - AWS Documentation:
-  - https://docs.aws.amazon.com/IAM/latest/UserGuide/access_policies.html
-  - https://docs.aws.amazon.com/IAM/latest/UserGuide/reference_policies.html
+  - [https://docs.aws.amazon.com/IAM/latest/UserGuide/access_policies.html](https://docs.aws.amazon.com/IAM/latest/UserGuide/access_policies.html)
+  - [https://docs.aws.amazon.com/IAM/latest/UserGuide/reference_policies.html](https://docs.aws.amazon.com/IAM/latest/UserGuide/reference_policies.html)
```
Comment (mariux) [inline README.md:190]: This is redundant please revert

File: examples/README.md [hunk 2]
```diff
@@ -1,9 +1,12 @@
-[<img src="https://raw.githubusercontent.com/mineiros-io/brand/master/mineiros-vertial-logo-smaller-font.svg" width="200"/>](https://mineiros.io/?ref=terraform-aws-iam-policy)
+[<img src="https://raw.githubusercontent.com/mineiros-io/brand/master/mineiros-primary-logo.svg" width="400"/>](https://mineiros.io/?ref=terraform-aws-iam-policy)
 
-[![Maintained by Mineiros.io](https://img.shields.io/badge/maintained%20by-mineiros.io-f32752.svg)](https://mineiros.io/?ref=terraform-aws-iam-policy)
-[![Terraform Version](https://img.shields.io/badge/terraform-~%3E%200.12.20-brightgreen.svg)](https://github.com/hashicorp/terraform/releases)
-[![License](https://img.shields.io/badge/License-Apache%202.0-brightgreen.svg)](https://opensource.org/licenses/Apache-2.0)
+[![Build Status](https://mineiros.semaphoreci.com/badges/terraform-aws-iam-policy/branches/master.svg?style=shields)](https://mineiros.semaphoreci.com/projects/terraform-aws-iam-policy)
```
Comment (mariux) [inline examples/README.md:3]: Build status on examples does not make sense unless we actually build them.. also version badge does not make much sense here

Comment (soerenmartius) [reply examples/README.md]: makes sense, fixed in 674c28b

File: Makefile [hunk 3]
```diff
@@ -57,8 +57,8 @@ test/pre-commit:
 test/unit-tests: DOCKER_FLAGS += ${DOCKER_SSH_FLAGS}
 test/unit-tests: DOCKER_FLAGS += ${DOCKER_AWS_FLAGS}
 test/unit-tests:
-	@echo "${YELLOW}[TEST] ${GREEN}Start Running Go Tests in Docker Container.${RESET}"
-	$(call go-test,./test/...)
+	@echo "${YELLOW}No tests defined.${RESET}"
+#	$(call go-test,tests)
```
Comment (mariux) [inline Makefile:62]: please do not deactivate already running tests.

File: Makefile [hunk 4]
```diff
@@ -82,6 +82,6 @@ help:
 quiet-command = $(if ${V},${1},$(if ${2},@echo ${2} && ${1}, @${1}))
 
 docker-run    = $(call quiet-command,${DOCKER_RUN_CMD} ${1} | cat,"${YELLOW}[DOCKER RUN] ${GREEN}${1}${RESET}")
-go-test       = $(call quiet-command,${DOCKER_RUN_CMD} go test -v -count=1 -timeout 45m -parallel 128 ${1} | cat,"${YELLOW}[TEST] ${GREEN}${1}${RESET}")
+go-test       = $(call quiet-command,${DOCKER_RUN_CMD} go test -v -timeout 45m -parallel 128 ${1} | cat,"${YELLOW}[TEST] ${GREEN}${1}${RESET}")
```
Comment (mariux) [inline Makefile:85]: you proposed to put this in ;) so please decide if it should be in or not.

File: examples/iam-policy-s3-full-access/README.md [hunk 5]
```diff
@@ -113,18 +138,73 @@ can't guarantee that exactly these actions will be performed if
...
+1. Use the `MAJOR` version for incompatible changes.
+1. Use the `MINOR` version when adding functionality in a backwards compatible manner.
+1. Use the `PATCH` version when introducing backwards compatible bug fixes.
+
+### Backwards compatibility in `0.0.z` and `0.y.z` version
+
+- In the context of initial development, backwards compatibility in versions `0.0.z` is **not guaranteed** when `z` is
+  increased. (Initial development)
+- In the context of pre-release, backwards compatibility in versions `0.y.z` is **not guaranteed** when `y` is
+  increased. (Pre-release)
```
Comment (mariux) [inline examples/iam-policy-s3-full-access/README.md:175]: This is not important for the example..

File: examples/iam-policy-s3-full-access/README.md [hunk 6]
```diff
@@ -113,18 +138,73 @@ can't guarantee that exactly these actions will be performed if
...
+
+Contributions are always encouraged and welcome! For the process of accepting changes, we use
+[Pull Requests](https://github.com/mineiros-io/terraform-aws-iam-policy/pulls). If you'd like more information, please
+see our [Contribution Guidelines](https://github.com/mineiros-io/terraform-aws-iam-policy/blob/master/CONTRIBUTING.md).
+
+## Makefile Targets
+
+This repository comes with a handy
+[Makefile](https://github.com/mineiros-io/terraform-aws-iam-policy/blob/master/Makefile).
+Run `make help` to see details on each available target.
```
Comment (mariux) [inline examples/iam-policy-s3-full-access/README.md:203]: There is no Makefile in this example.

Comment (soerenmartius) [reply examples/iam-policy-s3-full-access/README.md]: fixed in ee84401

File: examples/iam-policy-s3-full-access/README.md (hunk 5 above)
Comment (soerenmartius) [reply examples/iam-policy-s3-full-access/README.md]: 874a160

File: Makefile (hunk 4 above)
Comment (soerenmartius) [reply Makefile]: good catch, should defo stay

File: Makefile (hunk 3 above)
Comment (soerenmartius) [reply Makefile]: activated again

**Comment by soerenmartius** [comment]
once again, r4r @mariux

**Comment by soerenmartius** [comment]
r4r @mariux

File: examples/README.md [hunk 7]
```diff
@@ -1,9 +1,15 @@
+<<<<<<< HEAD
 [<img src="https://raw.githubusercontent.com/mineiros-io/brand/3bffd30e8bdbbde32c143e2650b2faa55f1df3ea/mineiros-primary-logo.svg" width="200"/>](https://mineiros.io/?ref=terraform-aws-iam-policy)
+=======
+[<img src="https://raw.githubusercontent.com/mineiros-io/brand/master/mineiros-primary-logo.svg" width="400"/>](https://mineiros.io/?ref=terraform-aws-iam-policy)
+>>>>>>> 6002e1c... correct broken README.mds
```
Comment (mariux) [inline examples/README.md:5]: conflicts in files not resolved

Comment (soerenmartius) [reply examples/README.md]: resolved in 53d269e. Updated to permanent logo also.

**Pull request merged**